src/style.css

```
@@ -0,0 +1,84 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+  flex-basis: 33.3%;
+  height: 300px;
+  color: white;
+}
+
+.block--1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+  order: 3;
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+  order: 2;
+}
+
+.block--6 {
+  background-color: rgb(0, 0, 0);
+  order: 1;
+}
+
+@media (max-width: 899px) {
```

That is more obvious to create a media query this way
```suggestion
@media (min-width: 900px) {
```